src/lib/services/feature-toggle-service.ts

```diff
@@ -1226,7 +1222,7 @@ class FeatureToggleService {
             segments: [],
             title: strategy.title,
             disabled: strategy.disabled,
-            // FIXME: Should we return sortOrder here, or adjust OpenAPI?
```

Comment:
@Tymek feature strategy schema has sortOrder so not sure what needs to be adjusted